src/Sylius/Bundle/AdminBundle/Menu/AdministratorStatusMenuBuilder.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Menu;
+
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use Sylius\Bundle\AdminBundle\Event\AdministratorStatusMenuBuilderEvent;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+final class AdministratorStatusMenuBuilder
```

Comment:
Why do we need this class?

Reply:
This class adds a Menu Builder to the Administrator status page so someone could add buttons like `Clear cache`